### ISSUE TITLE
Generalize method of sending aria-live alerts

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#165]
It would be really nice if we had a general way of sending aria-live alerts to the screen reader.  This would greatly reduce references to ```document``` in sim code and also isolate for maintenance and screen reader specific workarounds.

**Comment by jessegreenberg** [comment]
We could have something that collects all live elements in the PDOM and then provides an interface for passing along updated content to them.  

```js
document.getElementById( 'live-alert' ).textContent = this.getPositionDescription();
```

Would be simplified to something like
```js
var herald = new AriaHerald();
herald.alertAssertive( this.getPositionDescription() );
```

This is more phet-like too.

**Comment by jessegreenberg** [comment]
Above is a template with stubs for a Herald type that will send updates to the AT.

**Comment by jessegreenberg** [comment]
Working very well, this should be helpful.  Closing.
